packages/tab-bar/index.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from 'react';
+import classnames from 'classnames';
+// https://github.com/material-components/material-components-web-react/issues/533
+// @ts-ignore
+import TabScroller from '@material/react-tab-scroller';
+// https://github.com/material-components/material-components-web-react/issues/534
+// @ts-ignore
+import Tab from '@material/react-tab';
+// No mdc .d.ts files
+// @ts-ignore
+import {MDCTabBarFoundation} from '@material/tab-bar/dist/mdc.tabBar';
+
+export interface TabBarProps extends React.HTMLProps<HTMLDivElement> {
+  indexInView?: number;
+  activeIndex: number;
+  handleActiveIndexUpdate?: (index: number) => void;
+  className?: string;
+  isRtl?: boolean;
+};
+
+interface TabBarState {
+  previousActiveIndex: any;
+};
+
+export default class TabBar extends React.Component<
+  TabBarProps, TabBarState
+  > {
+  tabBarElement_: React.RefObject<HTMLDivElement> = React.createRef();
```

Comment:
Can these be private?

Reply:
This is also referenced in `/test/unit/tab-bar/index.test.tsx`.